src/components/PageHeaderNew/PageHeaderNew.scss

```diff
@@ -0,0 +1,53 @@
+.page-header-new {
+   margin: 0 auto;
+   padding: 24px 0 0;
+
+   &__text-container {
+      padding: 0 15px;
+      box-sizing: border-box;
+      width: 100%;
+      margin: auto;
+
+      @media screen and (min-width: 768px) {
+         display: flex;
+         max-width: 1048px;
+         padding: 0 48px;
+         align-items: center;
+         box-sizing: border-box;
+      }
+   }
+
+   &__title {
```

Comment:
If I make the text longer in the title or subtitle, the spacing changes. In order to keep it the same with any length of text, can you add width: 50% to the title and subtitle?